File: client/src/app/shared/models/orderItemBriefDto.ts
import { IOrder } from "./order";

export interface IOrderItemBriefDto
{
     checked: boolean;
     orderNo: number;
     orderDate: Date;
     customerName: string;
     orderItemId: number;
     requireInternalReview: boolean;
     categoryRef: string;
     categoryName: string;
     categoryRefAndName: string;
     quantity: number;
     status: string;
     assessmentQDesigned: boolean;
}

export class OrderItemBriefDto implements IOrderItemBriefDto
{
     checked: boolean = false;
     orderNo: number;
     orderDate: Date;
     customerName: string;
     orderItemId: number;
     requireInternalReview: boolean;
     categoryId: number;
     categoryRef: string;
     categoryName: string;
     categoryRefAndName: string;
     quantity: number;
     status: string;
     assessmentQDesigned: boolean;
}

File: core/ParamsAndDtos/OrderItemBriefDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace core.ParamsAndDtos
{
    public class OrderItemBriefDto
    {
        public int OrderItemId { get; set; }
        public int OrderId { get; set; }
        public string CustomerName { get; set; }
        public DateTime OrderDate { get; set; }
        public int CategoryId { get; set; }
        public string CategoryName { get; set; }
        public string CategoryRef {get; set;}
        public string CategoryRefAndName {get; set;}
        public int Quantity { get; set; }
        public int Status { get; set; }
    }
}